test/e2e/shim-install.sh

```diff
@@ -6,22 +6,20 @@ set -ex
 
 # Build gvisor-containerd-shim
 if [ "${INSTALL_LATEST}" === "1" ]; then
```

Comment:
This should just be a == I think. Currently it shows an error in travis

    if [ "${INSTALL_LATEST}" == "1" ]; then

Reply:
Done